### PR TITLE
Add fees for Hipo

### PR DESCRIPTION
**Pull request by behrang**

### CODE REVIEW

**Comment by llamatester** [comment]
The hipo adapter exports: 
        
 
 _______________________________________
    Fees for 5/11/2023
    _______________________________________
    
    TON 👇
    Backfill start time: 30/10/2023
    Daily supply side revenue: {
      "coingecko:the-open-network": "123.492452514"
    }
    └─ Methodology: Stakers receive the rest of the rewards, after deducting validators share and protocol fee.
    Daily holders revenue: {
      "coingecko:the-open-network": "0.000000000"
    }
    └─ Methodology: Currently there is not government token.
    Daily protocol revenue: {
      "coingecko:the-open-network": "8.232964167"
    }
    └─ Methodology: Hipo receives a small fee before distributing rewards to stakers.
    Daily revenue: {
      "coingecko:the-open-network": "8.232964167"
    }
    └─ Methodology: All generated revenue is from protocol fee.
    Daily user fees: {
      "coingecko:the-open-network": "0.000000000"
    }
    └─ Methodology: Stakers pay no fees for using Hipo.
    Daily fees: {
      "coingecko:the-open-network": "131.725416681"
    }
    └─ Methodology: The total reward is calculated after deducting validators share, so it is the stakers revenue plus protocol revenue.
    
    
    
    
    Done in 5.59s.

File: fees/hipo/index.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                })
+                if (!response1.data.ok)  {
+                    throw new Error('Error in calling toncenter.com/api/v2/runGetMethod')
+                }
+                const getTreasuryState = response1.data.result
+                if (getTreasuryState.exit_code !== 0) {
+                    throw new Error('Expected a zero exit code, but got ' + getTreasuryState.exit_code)
+                }
+
+                await sleep(1000)
```
Comment (dtmkeng) [inline fees/hipo/index.ts:38]: have question which issue why should use `delay` ?

Comment (behrang) [reply fees/hipo/index.ts:38]: Because of a limitation of 1 request per second on toncenter.com free calls.

Comment (dtmkeng) [reply fees/hipo/index.ts:38]: can i remove ?

**Pull request merged**